src/components/page-environment/environment-overview.js

```diff
@@ -119,7 +119,7 @@ export class EnvironmentOverview extends React.Component {
         >
           {loaded && (
             <div className="env__content">
-              <section className="grid">
+              <section className="grid grid--gap-medium">
```

Comment:
add class `component__overview` to parent div in order to get two columns in the overview section. This class is used multiple places to it may be a good idea to give it a more universal name. 
```suggestion
              <section className="grid grid--gap-medium component__overview">
```

Reply:
this class is a part of the `page-active-component` and not in true global space. If we want to use it elsewhere it should become global instead